core/org.wso2.carbon.core.services/src/main/java/org/wso2/carbon/core/services/util/CarbonAuthenticationUtil.java

```diff
@@ -241,6 +241,22 @@ public static void onSuccessAdminLogin(ThriftSession thriftSession, String usern
                 .getConfigSystemRegistry(tenantId);
         loginSubscriptionManagerServiceImpl.triggerEvent(configRegistry, username, tenantId,tenantDomain);
 
+
+
+        // Load tenant : This is needed because we have removed ActivationHandler,
+        // which did the tenant loading part earlier with login. So we load tenant after successful login
+        try {
+            if (!MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
+                TenantAxisUtils.getTenantConfigurationContext(tenantDomain,
+                        CarbonServicesServiceComponent.
+                                getConfigurationContextService().
+                                getServerConfigContext());
+            }
+        } catch (Exception e) {
+            log.error("Error trying load tenant after successful login", e);
```

Comment:
@daneshk - Don't we need to Throw the exception? Here we are only logging the Exception.